src/main/java/de/doubleslash/keeptime/model/repos/UserRepository.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 doubleSlash Net Business GmbH
+//
+// This file is part of KeepTime.
+// KeepTime is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package de.doubleslash.keeptime.model.repos;
+
+import de.doubleslash.keeptime.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, String> {
```

Comment:
can we remove all the extra logic with repositories and database and use the mechanism spring offers in application.properties. to define user and password as properties and not from database. would make it a lot easier to maintain.

Reply:
Done.